test/components/xDragBoxLayerTests.ts

```diff
@@ -1,178 +1,155 @@
-      assert.strictEqual(actualBounds.bottomRight.y, dbl.height(), "box ends at bottom");
-      assert.strictEqual(actualBounds.bottomRight.x, bottomRight.x, "right edge set correctly");
-
-      svg.remove();
-    });
-
-    it("resizes only in x", () => {
-      let svg = TestMethods.generateSVG(SVG_WIDTH, SVG_HEIGHT);
-      let dbl = new Plottable.Components.XDragBoxLayer();
-      dbl.boxVisible(true);
-      dbl.resizable(true);
-      dbl.renderTo(svg);
-
-      let topLeft = {
+      let quarterTopLeftPoint = {
         x: SVG_WIDTH / 4,
         y: SVG_HEIGHT / 4
       };
-      let bottomRight = {
+      let middlePoint = {
         x: SVG_WIDTH / 2,
         y: SVG_HEIGHT / 2
       };
-
-      dbl.bounds({
-        topLeft: topLeft,
-        bottomRight: bottomRight
-      });
-
-      let actualBounds = dbl.bounds();
-      let dragTo = {
+      let quarterBottomRightPoint = {
         x: SVG_WIDTH * 3 / 4,
-        y: SVG_HEIGHT / 2
+        y: SVG_HEIGHT * 3 / 4
       };
-      TestMethods.triggerFakeDragSequence(dbl.background(), actualBounds.bottomRight, dragTo);
-      actualBounds = dbl.bounds();
-      assert.strictEqual(actualBounds.bottomRight.x, dragTo.x, "resized in x");
-      assert.strictEqual(actualBounds.topLeft.y, 0, "box still starts at top");
-      assert.strictEqual(actualBounds.bottomRight.y, dbl.height(), "box still ends at bottom");
-      svg.remove();
-    });
 
-    it("stays full height after resizing", () => {
-      let svg = TestMethods.generateSVG(SVG_WIDTH, SVG_HEIGHT);
-      let dbl = new Plottable.Components.XDragBoxLayer();
-      dbl.boxVisible(true);
-      dbl.resizable(true);
-      dbl.renderTo(svg);
+      let svg: d3.Selection<void>;
+      let dbl: Plottable.Components.XDragBoxLayer;
 
-      let topLeft = {
-        x: SVG_WIDTH / 4,
-        y: SVG_HEIGHT / 4
-      };
-      let bottomRight = {
-        x: SVG_WIDTH / 2,
-        y: SVG_HEIGHT / 2
-      };
-
-      dbl.bounds({
-        topLeft: topLeft,
-        bottomRight: bottomRight
+      beforeEach(() => {
+        svg = TestMethods.generateSVG(SVG_WIDTH, SVG_HEIGHT);
+        dbl = new Plottable.Components.XDragBoxLayer();
       });
 
-      let heightBefore = dbl.height();
-      let boundsBefore = dbl.bounds();
-      svg.attr("height", 2 * SVG_HEIGHT);
-      dbl.redraw();
-      assert.notStrictEqual(dbl.height(), heightBefore, "component changed size");
-
-      let boundsAfter = dbl.bounds();
-      assert.strictEqual(boundsAfter.topLeft.x, boundsBefore.topLeft.x, "box keeps same left edge");
-      assert.strictEqual(boundsAfter.topLeft.y, 0, "box still starts at top");
-      assert.strictEqual(boundsAfter.bottomRight.x, boundsBefore.bottomRight.x, "box keeps same right edge");
-      assert.strictEqual(boundsAfter.bottomRight.y, dbl.height(), "box still ends at bottom");
-      svg.remove();
-    });
+      it("has the correct bounds()", () => {
```

Comment:
Maybe something more specific, like "has bounds() that extend across its full width"?